src/caselawclient/xquery/insert_document.xqy

```diff
@@ -0,0 +1,15 @@
+xquery version "1.0-ml";
+
+import module namespace dls = "http://marklogic.com/xdmp/dls" at "/MarkLogic/dls.xqy";
+
+declare variable $uri as xs:string external;
+declare variable $document as xs:string external;
+
+let $document_xml := xdmp:unquote($document)
+
+let $collections :=
+  if (fn:contains($uri, 'press_summary'))
```

Comment:
we decided that we will have `/press_summary/` in the uri, so this is a good pattern to look for to distinguish the doc types on ingestion

Reply:
This makes sense for now; I'd like to start relying on a structured data bundle arriving from the parser in future so we can move all the things like failure states into there as well, but that's a future step.